src/app/containers/FeaturesAnalysis/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,293 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CpsRelatedContent should render Story Feature components in Live environment 1`] = `
+<DocumentFragment>
+  <section
+    aria-labelledby="features-analysis-heading"
+    class="GhostGrid-sc-12lwanc-1 kGasFS"
+    role="region"
+  >
+    <div
+      class="GridItemConstrainedLarge-sc-12lwanc-5 Wrapper-r0jyv5-0 ftAujp"
+    >
+      <div
+        class="GridItemConstrainedLarge-sc-12lwanc-5 Wrapper-r0jyv5-0 ftAujp"
+      >
+        <div
+          class="SectionLabelWrapper-sc-1x4gwjz-1 gKFUvc StyledSectionLabel-r0jyv5-1 dRpDSS"
+        >
+          <div
+            class="Bar-sc-1x4gwjz-0 jsgdXU"
+          />
+          <h2
+            class="Heading-sc-1x4gwjz-2 eZofTG"
+          >
+            <div
+              class="FlexColumn-wx5kdp-0 dgpsiS"
+            >
+              <span
+                class="FlexRow-wx5kdp-2 dVBNyZ"
+              >
+                <span
+                  class="Title-wx5kdp-4 jiurmA"
+                  dir="ltr"
+                  id="features-analysis-heading"
+                >
+                  Features Analysis 
+                </span>
+              </span>
+            </div>
+          </h2>
+        </div>
+        <ul
+          class="StoryPromoUl-sc-171lqjd-1 isCYWc"
+          role="list"
+        >
+          <li
+            class="StoryPromoLi-sc-171lqjd-0 dqJaJG"
+            role="listitem"
+          >
+            <div
+              class="StoryPromoWrapper-sc-1dvfmi3-0 buBJys"
+            >
+              <div
+                class="ImageGridItem-sc-1xk56cj-0 fNYKlW"
+                dir="ltr"
+              >
+                <div
+                  class="ImageContentsWrapper-sc-1dvfmi3-1 cinwNl"
+                >
+                  <div
+                    class="ImagePlaceholder-sc-11u25v2-0 fNqlUz"
+                  >
+                    <div
+                      class="lazyload-placeholder"
```

Comment:
should we have an example of an image with alt text??

Reply:
This resolves to something like:
```
<img 
srcset="https://ichef.bbci.co.uk/news/70/cpsdevpb/6ACE/test/_63724372_gettyimages-1098075358.jpg 70w, https://ichef.bbci.co.uk/news/95/cpsdevpb/6ACE/test/_63724372_gettyimages-1098075358.jpg 95w, https://ichef.bbci.co.uk/news/144/cpsdevpb/6ACE/test/_63724372_gettyimages-1098075358.jpg 144w, https://ichef.bbci.co.uk/news/183/cpsdevpb/6ACE/test/_63724372_gettyimages-1098075358.jpg 183w, https://ichef.bbci.co.uk/news/240/cpsdevpb/6ACE/test/_63724372_gettyimages-1098075358.jpg 240w, https://ichef.bbci.co.uk/news/320/cpsdevpb/6ACE/test/_63724372_gettyimages-1098075358.jpg 320w, https://ichef.bbci.co.uk/news/624/cpsdevpb/6ACE/test/_63724372_gettyimages-1098075358.jpg 624w" 
alt="Some people at the NTA awards" 
src="https://ichef.bbci.co.uk/news/660/cpsdevpb/6ACE/test/_63724372_gettyimages-1098075358.jpg" 
sizes="(max-width: 1008px) 33vw, 237px" 
width="976" 
class="StyledImg-sc-7vx2mr-0 hIxkbt">
```